distributed/tests/test_client.py

```diff
@@ -7262,6 +7263,9 @@ def fetch_comp_code(dask_scheduler):
 
 def test_computation_object_code_not_available(client):
```

Comment:
I'm not sure what the original motivation for this tests is. Is it `numpy`-specific? If not, maybe we should refactor the test so that it still runs when using the latest version of `numpy`

Reply:
I am not aware of other use cases where the stack is not available. I may, of course, be wrong.